helm/botkube/templates/_helpers.tpl

```diff
@@ -41,3 +41,7 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{- define "botkube.CommunicationsSecretName" -}}
+{{- default (include "botkube.fullname" .) (.Values.communications.existingSecretName) -}}
```

Comment:
I think the default name should be kept what it was before i.e. `{{ include "botkube.fullname" . }}-communication-secret`. That way the name will indicate correctly that it is a communication related secret. Also, we won't break anything if someone is depending on that name to do something (less likely, but still).